docs/application-connector/docs/02-01-application-connector.md

```diff
@@ -10,19 +10,19 @@ type: Architecture
 
 The Nginx Ingress Controller exposes the Application Connector by assigning a public IP address and a DNS name to it.
 The DNS name of the Ingress is cluster-dependant and follows the `gateway.{cluster-dns}` format, for example `gateway.servicemanager.cluster.kyma.cx`.
-You can access every exposed Application (App) through its Gateway by using the assigned path. For example, to reach the Gateway for the `user-custom` App, use this URL: `gateway.servicemanager.cluster.kyma.cx/user-custom`. The Nginx Ingress Controller secures the endpoint with certificate validation. Each call must include a valid client certificate which is App-specific.
+You can access every exposed Application (App) through its gateway by using the assigned path. For example, to reach the gateway for the `user-custom` App, use this URL: `gateway.servicemanager.cluster.kyma.cx/user-custom`. The Nginx Ingress Controller secures the endpoint with certificate validation. Each call must include a valid client certificate which is App-specific.
 
 ## Connector Service
 
 The Connector Service:
 - Handles the exchange of client certificates for a given RE.
 - Provides the Application Registry and Event Service endpoints.
-- Signs client certificates using the server side certificate stored in Kubernetes Secrets.
+- Signs client certificates using the server-side certificate stored in a Kubernetes Secret.
 
 ## Application Registry
 
-The Application Registry stores all registered APIs and Event Catalog exposed by a connected external solution. The APIs and Event catalogs metadata are stored in the [Application](#custom-resource-application) custom resource.
-The system creates a new Kubernetes service for each registered API. Additionally, a new Service Classes is registered in the Service Catalog.
+The Application Registry stores all registered APIs and Event Catalog exposed by a connected external solution. The metadata of the external solution's APIs and Event catalogs is stored in the [Application](#custom-resource-application) custom resource.
+The system creates a new Kubernetes service for each registered API. Additionally, new Service Classes is registered in the Service Catalog.
```

Comment:
```suggestion
The system creates a new Kubernetes service for each registered API. Additionally, new ServiceClasses are registered in the Service Catalog.
```
or ` a ServiceClass is`  - depending what is the meaning here, but reading the doc further I'd go with `ServiceClasses are`